ui/raidboss/data/07-dt/map/cenote_ja_ja_gural.ts

```diff
@@ -0,0 +1,108 @@
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+const cenoteJaJaOutputStrings = {
+  spawn: {
+    en: '${name} spawned!',
+    de: '${name} erscheint!',
+    fr: '${name} apparait !',
+    ja: '${name} 現れる！',
+    cn: '已生成 ${name}!',
+    ko: '${name} 등장!',
+  },
+} as const;
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  id: 'CenoteJaJaGural',
+  zoneId: ZoneId.CenoteJaJaGural,
+
+  triggers: [
+    // ---------------- random treasure mobs ----------------
+    {
+      id: 'Cenote Ja Ja Alpaca of Fortune Spawn',
+      // 13240 = Alpaca of Fortune
+      type: 'AddedCombatant',
+      netRegex: { npcNameId: '13240' },
+      suppressSeconds: 1,
+      infoText: (_data, matches, output) => output.spawn!({ name: matches.name }),
+      outputStrings: {
+        spawn: cenoteJaJaOutputStrings.spawn,
+      },
```

Comment:
I think you could just do this:

```suggestion
      outputStrings: cenoteJaJaOutputStrings,
```